docs/pages/docs/guide/advanced/table.mdx

```diff
@@ -171,7 +171,18 @@ Just wrap your table with curly braces `{` and `}`, e.g.
 </table>}
 ```
 
-#### Changing Default Behaviour
+Alternatively, create a reusable table component:
+
+```mdx filename="MDX"
+export function Table({ columns, data }) {
+  return <table>{/* ... */}</table>
+}
+
+<Table columns={...} data={...} />
+```
+
+{/* prettier-ignore */}
+{/* #### Changing Default Behaviour
```

Comment:
I just tested it locally and it worked
<img width="547" alt="image" src="https://github.com/user-attachments/assets/7460a5b6-65f5-4632-9b19-f324e05d4cdb">
<img width="270" alt="image" src="https://github.com/user-attachments/assets/9827aa88-9fa9-4f0f-97fb-42bed378424e">
<img width="1263" alt="image" src="https://github.com/user-attachments/assets/2bd938f0-74ab-44f2-ab08-5a16bc10d1e7">